fission/src/ui/components/MainHUD.tsx

```diff
@@ -127,6 +128,11 @@ const MainHUD: React.FC = () => {
                         onClick={() => openPanel("poker")}
                     />
                     <MainHUDButton value={"Test God Mode"} icon={<IoGameControllerOutline />} onClick={TestGodMode} />
+                    <MainHUDButton
+                        value={"Clear Prefs"}
+                        icon={<IoBug />}
+                        onClick={() => PreferencesSystem.clearPreferences()}
+                    />
```

Comment:
After I click this then my robot stops moving and the green preference sphere stays attached

Reply:
All this does is wipes preferences from local storage, but does not update any robots that are currently spawned. It's just for debugging purposes and you should reload the page after clearing prefs.

Reply:
No errors in the log and I can keep spawning other robots in